frontend/src/views/funding/FundingOverviewView.vue

```diff
@@ -54,47 +163,211 @@
 </template>
 
 <script>
+import AppButton from '@/components/ui/AppButton.vue'
 import OrganizationHeader from '@/components/organizations/OrganizationHeader.vue'
 import AppBackButton from '@/components/ui/AppBackButton.vue'
+import AppLabel from '@/components/ui/AppLabel.vue'
+import AppButtonRadioGroup from '@/components/ui/AppButtonRadioGroup.vue'
+import FundingAgreementService from '@/services/fundingAgreementService'
+import ApplicationService from '@/services/applicationService'
+import format from '@/utils/format'
+import { FUNDING_AGREEMENT_STATUS_CODES } from '@/utils/constants'
+import { mapState } from 'pinia'
+import { useAuthStore } from '@/stores/auth'
+import { ApiRoutes } from '@/utils/constants'
+import ApiService from '@/common/apiService'
+import moment from 'moment'
 
 export default {
   name: 'FundingOverviewView',
-  components: { AppBackButton, OrganizationHeader },
+  components: { AppButton, AppBackButton, AppLabel, AppButtonRadioGroup, OrganizationHeader },
   data() {
     return {
+      format,
       tab: null,
-      fundingAgreements: [
-        {
-          guid: '21d0d405-900c-ef11-9f89-000d3af44283',
-          faNumber: 'FA-123456',
-          faType: 'Base Funding',
-          facility: 'Facility 1',
-          startDate: 'Nov 22, 2023',
-          endDate: 'Nov 22, 2024',
-          status: 'FA Signature Pending',
-        },
-        {
-          guid: '21d0d405-900c-ef11-9f89-000d3af44283',
-          faNumber: 'FA-000002',
-          faType: 'Supplementary Allowances',
-          facility: 'Facility 2',
-          startDate: 'May 15, 2023',
-          endDate: 'May 15, 2024',
-          status: 'FA Submitted to Ministry',
-        },
-      ],
+      loading: false,
+      fundingAgreements: [],
+      orgUsers: [],
+      selectedFacility: null,
+      selectedDateFilterType: null,
+      selectedPaymentFilterTypes: [],
+      startDateFrom: null,
+      startDateTo: null,
+      menuStartDateFrom: false,
+      menuStartDateTo: false,
       headers: [
-        { title: 'FA Number', key: 'faNumber' },
-        { title: 'FA Type', key: 'faType' },
-        { title: 'Facility', key: 'facility' },
+        { title: 'Funding Agreement Number', key: 'fundingAgreementNumber' },
+        { title: 'Funding Agreement Type', key: 'fundingAgreementType' },
+        { title: 'Facility Name', key: 'facility' },
+        { title: 'Signing Expense Authority', key: 'expenseAuthorityName' },
         { title: 'Start Date', key: 'startDate' },
         { title: 'End Date', key: 'endDate' },
         { title: 'Status', key: 'status' },
         { title: 'Actions', key: 'actions' },
       ],
+      dateFilterTypes: [
+        { label: '3 Months', value: '3Months' },
+        { label: '6 Months', value: '6Months' },
+        { label: 'YTD', value: 'YTD' },
+        { label: 'Custom', value: 'Custom' },
```

Comment:
These values are repeated on line 239 (and elsewhere) so they should be constants. If it's just screen specific and not used in the template you can define it in the script but outside of the component.

Reply:
Fixed.

Reply:
If your constants don't need to be access in the template you can just define them outside. See LOCKED_STATUSES in FundingView.vue. It'll keep your code separation cleaner.

Reply:
Still outstanding.

Reply:
Thanks for extra description and reference.... i think i got it now.